pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -444,6 +447,14 @@ func (r *ReconcileHyperConverged) ensureKubeVirt(instance *hcov1alpha1.HyperConv
 
 	logger.Info("KubeVirt already exists", "KubeVirt.Namespace", found.Namespace, "KubeVirt.Name", found.Name)
 
+	if !reflect.DeepEqual(found.Spec, virt.Spec) {
+		if found.Spec.UninstallStrategy == "" {
+			logger.Info("Updating UninstallStrategy on existing KubeVirt to its default value")
+			found.Spec.UninstallStrategy = virt.Spec.UninstallStrategy
```

Comment:
Layman's question: What if the user changed the `UninstallStrategy` on purpose to the non-default value?
Will the reconcile-loop put the default value back again?

Reply:
No, because in that case found.Spec.UninstallStrategy will be != "".
We are explicitly setting it only if empty.